### ISSUE TITLE
Revert recent case twiddling?

### ISSUE THREAD

**Issue by nckx** [sarabander/sicp#23]
Thank you for a wonderful, portable version of SICP!

I noticed two commits in the past year that change two (single) occurrences of symbols beginning a sentence to lowercase:

- 68dd959 changed `else` keyword to lower case
- 382b325 Fix [:disappointed:] `good-enough?` case

However, capitalisation is done throughout the book:

- “`Not` is an ordinary procedure.”
- “`Average-damp` is a procedure [...] Using `average-damp`, we can [...]”

Scheme itself isn't case-sensitive, so this makes sense. There are implementations that are. I can see how someone might have typed `Else` into their REPL, got an error thrown back, and fired off a well-meaning PR.

Even so, going against the obvious intentions of the authors of both the book and the language is quite... invasive.

If this version is considered to be a modernised rendering of the original, and not an editorial fork, I'd like to request reverting the above changes to match both ‘upstream’ and the rest of the book.

**Comment by sarabander** [comment]
Good point. I guess it is better to revert the changes for consistency. On the other hand, I accepted a large patch of changes to sicp-pdf a while ago that uncapitalized the symbols very thoroughly. Nevertheless, I would err on the direction of being faithful to the original. 

Thank your for raising the issue, I will make the suggested change soon.

**Comment by nckx** [comment]
Thanks for your response!

> On the other hand, I accepted a large patch of changes to sicp-pdf a while ago that uncapitalized the symbols very thoroughly.

Oh, sure! It's different if done thoroughly and consistently. My main issue was with 2 commits changing 2 (seemingly) random occurrences for unclear reasons, and leaving all others.

I guess most people expect programming languages to be case-sensitive nowadays. While I admit to preferring the original old-school style, I do understand why one might make a different choice.

> Nevertheless, I would err on the direction of being faithful to the original.

I think that's what most readers expect from this version — but again, that's my biased opinion...
